src/sentry/static/sentry/app/views/dashboards/utils/getDiscoverUrlPathFromDiscoverQuery.jsx

```diff
@@ -38,3 +40,70 @@ export function getDiscoverUrlPathFromDiscoverQuery({organization, selection, qu
     limit: 1000,
   })}&visualization=${visual}`;
 }
+
+export function getDiscover2UrlPathFromDiscoverQuery({
+  organization,
+  selection,
+  query: d1Query,
+}) {
+  const d2Query = {
+    name: d1Query.name,
+    field: ['title', ...d1Query.fields],
+    sort: d1Query.orderby,
+    statsPeriod: selection?.datetime?.period,
+  };
+
+  const queryQueries = (d1Query.conditions || []).map(c => {
+    const tag = c[0] || '';
+    const val = c[2] || '';
+
+    const operator = c[1] || '';
+    const isNot = operator.includes('!') || operator.includes('NOT');
+    const isNull = operator.includes('NULL');
+    const isLike = operator.includes('LIKE') || operator.includes('=');
```

Comment:
Won't this make old `=` conditions into wildcards based on line 86?

Reply:
I was under the impression that `=` was equivalent to `LIKE`, but I've clarified that on Slack. Good catch, thanks.